sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -713,9 +713,14 @@ func (host *nodeLanguageHost) InstallDependencies(
 	// best effort close, but we try an explicit close and error check at the end as well
 	defer closer.Close()
 
+	ctx := server.Context()
+
+	tracingSpan, _ := opentracing.StartSpanFromContext(ctx, "npm-install")
+	defer tracingSpan.Finish()
+
 	stdout.Write([]byte("Installing dependencies...\n\n"))
 
-	_, err = npm.Install(server.Context(), req.Directory, false /*production*/, stdout, stderr)
+	_, err = npm.Install(ctx, req.Directory, false /*production*/, stdout, stderr)
```

Comment:
I think the ignored argument on line 718 (a `context.Context`) should be passed in to `npm.Install` on line 723, right? I presume `opentracing.StartSpanFromContext` subsumes the context provided to produce a new context [tagged with span information](https://pkg.go.dev/github.com/opentracing/opentracing-go#StartSpanFromContext). This allows a new child context inside `npm.Install` to correctly point to `tracingSpan` as its parent.